dyno/panda/dyno_panel.py

```diff
@@ -0,0 +1,133 @@
+import tkinter as tk
+from tkinter import font
+from PIL import Image, ImageTk
+import serial
+import time
+
+#Define constants
+NUMSENSORS1 = 2 #RPM AND TEMPERATURE
+NUMSENSORS2 = 2 #JOULEMETER
+#---------------------
+
+#Images
+dino_path = "./img/dino.gif"
+start_path = "./img/start.png"
+#---------------------
+
+#Colors
+color_bg = "#16495C"
+color_fg = "#DA8669"
+color_label = "#B1D9DB"
+#---------------------
+
+#Placement
+LABEL_COL1_PLACE = {'relx':0.15, 'rely':0.6}
+LABEL_COL2_PLACE = {'relx':0.55, 'rely':0.6}
+LABEL_SPACE = 0.08
+#---------------------
```

Comment:
Not that it's necessary now, but eventually it might be nice to separate the frontend from the backend here into separate files. Feel free to just mark this as resolved for now, just a thing to keep in mind.

Reply:
Yeah I was thinking about this too. Is not this entire script just frontend? I guess the functions which print and read serial can be moved to a separate script?